types/spritesmith/index.d.ts

```diff
@@ -0,0 +1,60 @@
+// Type definitions for spritesmith 3.4
+// Project: https://github.com/twolfson/spritesmith
+// Definitions by: Zenoo <https://github.com/Zenoo>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { ReadableStream } from 'stream/web';
+import { BufferFile } from 'vinyl';
+
+interface SpritesmithOptions {
```

Comment:
they call this 'Params' for some reason and are using all other the readme:
https://github.com/twolfson/spritesmith#new-spritesmithparams

Reply:
I changed it to SpritesmithParams, although I don't really see why that's important ^^

Reply:
just sticking with original intent/naming, the name does not really matter in TS, if shape of type is same